### PR TITLE
PR: Use `logging` module instead of `print`

### LINKED ISSUE #27: Prevent showing `print` statements when running programmatically
Some options:

* Use logging module instead of plain `print` statements
* Add flag to only use print if requested (for example when calling things from the CLI)

### PR DESCRIPTION
**Pull request by dalthviz**
Fixes #27

### CODE REVIEW

File: envs_manager/cli.py [hunk 1]
```diff
@@ -148,10 +155,19 @@ def main(args=None):
...
-    print(f"Using ENV_BACKEND: {options.backend}")
-    print(f"Using ENV_BACKEND_EXECUTABLE: {EXTERNAL_EXECUTABLE}")
+
+    # Setup logging
+    logging.basicConfig(
+        level=options.logging_level,
+        format="%(message)s",
+        handlers=[logging.StreamHandler(stream=sys.stdout)],
+    )
+    logger = logging.getLogger("envs-manager")
```
[the remark this answers is not in the document]
Comment (dalthviz) [reply envs_manager/cli.py:165]: Without that you will not get output for some operations that show the subprocess captured output (for example you will not see an env list of packages)

File: envs_manager/backends/conda_like_interface.py [hunk 2]
```diff
@@ -3,18 +3,21 @@
...
 import yaml
-import requests
 
 from envs_manager.api import EnvManagerInstance, run_command, get_package_info
 
 MICROMAMBA_VARIANT = "micromamba"
 CONDA_VARIANT = "conda"
 
 
+logger = logging.getLogger("envs-manager")
```
[the remark this answers is not in the document]
Comment (dalthviz) [reply envs_manager/backends/conda_like_interface.py:18]: I just put there a name that was easy for me to spot while checking other stuff, should I change it?

File: envs_manager/cli.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dalthviz) [reply envs_manager/cli.py:165]: Also, this is only done for the CLI/when running from the CLI

File: envs_manager/backends/conda_like_interface.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ccordoba12) [reply envs_manager/backends/conda_like_interface.py:18]: Could you post how the logging output looks like with `env-manager` and `__name__` for a single file? That way we can take a more informed decision.

[the remark this answers is not in the document]
Comment (dalthviz) [reply envs_manager/backends/conda_like_interface.py:18]: Using the option Spyder `debug-info` `minimal` flag you get:

* `envs-manager`
image

* `__name__`:
image

If you use the CLI, the logger is formated to only show the messages, so you see:

image

[the remark this answers is not in the document]
Comment (ccordoba12) [reply envs_manager/backends/conda_like_interface.py:18]: Ok, if this necessary to have the the right output in `subprocess` calls and not to debug the package, then I'd say your choice for the logger name is fine.

**Review by ccordoba12 — COMMENTED** [review]
Thanks @dalthviz!

**Pull request merged**